### ISSUE TITLE
different return codes for ERR_CONNECTING_IN_PROGRESS

### ISSUE THREAD

**Issue by SF-DI** [tanelih/phonegap-bluetooth-plugin#45]
Hi Taneli,

regarding the documentation
https://github.com/tanelih/phonegap-bluetooth-plugin/blob/master/manual/src/android/org/apache/cordova/bluetooth/BluetoothError.java
the return code for ERR_CONNECTING_IN_PROGRESS is 6.
In my code I get 5 instead of 6 in the connect error event.

{"message":"There is already a connection in progress.","code":5}

What's the reason?
Or belongs "There is already a connection in progress." to "ERR_CONNECTION_ALREADY_EXISTs"? I don't think so.

Regards,
Marco Diers

**Comment by tanelih** [comment]
Hi,

The `ERR_CONNECTING_IN_PROGRESS` should produce error message about a connection *attempt* in progress. The `ERR_CONNECTION_ALREADY_EXISTS` should produce a message about a connection already in progress... It's probably bad english from my part (not a native speaker), sorry!

TLDR; The behavior you're seeing is intended.
